stylo: Don't add boundary characters to the fixups specified in ServoBindings.toml

emilio opened the pull request:
This allows adjusting the fixups for the patches in bug 1393230 when using
libclang 3.9.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/servo/servo/18408)
<!-- Reviewable:end -->

highfive commented:
Heads up! This PR modifies the following files:
 * @bholley: components/style/build_gecko.rs
 * @canaltinova: components/style/build_gecko.rs

highfive commented:
<img src="http://www.joshmatthews.net/warning.svg" alt="warning" height=20> **Warning** <img src="http://www.joshmatthews.net/warning.svg" alt="warning" height=20>

* These commits modify style code, but no tests are modified. Please consider adding a test!

emilio committed:
stylo: Don't add boundary characters to the fixups specified in ServoBindings.toml

This allows adjusting the fixups for the patches in bug 1393230 when using
libclang 3.9.

emilio commented:
This was reviewed by @upsuper in the mentioned bug.

@bors-servo r=upsuper p=10

bors-servo commented:
:pushpin: Commit 86f4e22 has been approved by `upsuper`

<!-- @bors-servo r=upsuper 86f4e22fc2ead6a99f296d5086aa7845cc9464a8 -->

bors-servo commented:
:hourglass: Testing commit 86f4e22fc2ead6a99f296d5086aa7845cc9464a8 with merge 0c2abbd7eac17da99f9a7a5dc61623a87600f1da...

bors-servo commented:
:sunny: Test successful - [android](http://build.servo.org/builders/android/builds/8767), [arm32](http://build.servo.org/builders/arm32/builds/9000), [arm64](http://build.servo.org/builders/arm64/builds/9031), [linux-dev](http://build.servo.org/builders/linux-dev/builds/9091), [linux-rel-css](http://build.servo.org/builders/linux-rel-css/builds/5883), [linux-rel-wpt](http://build.servo.org/builders/linux-rel-wpt/builds/5930), [mac-dev-unit](http://build.servo.org/builders/mac-dev-unit/builds/9132), [mac-rel-css1](http://build.servo.org/builders/mac-rel-css1/builds/1591), [mac-rel-css2](http://build.servo.org/builders/mac-rel-css2/builds/1597), [mac-rel-wpt1](http://build.servo.org/builders/mac-rel-wpt1/builds/5827), [mac-rel-wpt2](http://build.servo.org/builders/mac-rel-wpt2/builds/5884), [mac-rel-wpt3](http://build.servo.org/builders/mac-rel-wpt3/builds/1615), [mac-rel-wpt4](http://build.servo.org/builders/mac-rel-wpt4/builds/1609), [windows-msvc-dev](http://build.servo.org/builders/windows-msvc-dev/builds/4386)
Approved by: upsuper
Pushing 0c2abbd7eac17da99f9a7a5dc61623a87600f1da to master...
